inc/networkportinstantiation.class.php

```diff
@@ -575,22 +615,32 @@ function showMacField(NetworkPort $netport, $options = []) {
 
 
    /**
-    * Display the Netpoint field. Used by Ethernet, and Migration
+    * Display the Socket field. Used by Ethernet, and Migration
     *
     * @param NetworkPort $netport         NetworkPort object :the port that owns this instantiation
     *                                     (usefull, for instance to get network port attributs
     * @param array       $options         array of options given to NetworkPort::showForm
     * @param array       $recursiveItems  list of the items on which this port is attached
    **/
-   function showNetpointField(NetworkPort $netport, $options = [], $recursiveItems = []) {
+   function showSocketField(NetworkPort $netport, $options = [], $recursiveItems = []) {
```

Comment:
Please add ```- `NetworkPortInstantiation::showNetpointField()` renamed to `NetworkPortInstantiation::showSocketField()`.``` in Changelog > API changes > Changes

Reply:
done